test/Parse/operator_decl.swift

```diff
@@ -117,3 +117,5 @@ infix operator *<*< : F, Proto
 
 // expected-error@+2 {{expected precedence group name after ':' in operator declaration}}
 postfix operator ++: // expected-error {{only infix operators may declare a precedence}} {{20-21=}}
+
+dynamic prefix operator +!+  // expected-error{{unexpected attribute 'dynamic' in operator declaration}}
```

Comment:
I see that you moved this test case. Could you keep the assertion that there should be a Fix-It to remove `dynamic`, i.e. the `{{1-9=}} part?

---

Could you add a newline at the end of the file?